src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -52,16 +53,61 @@ Hegel goes to add a link between the empty thinking (or empty intuiting) that fo
 
```

Comment:
This is a comment for line 50 - " However, there is nonetheless a difference here since empty thinking (or empty intuiting) is still _thinking_, and that in itself is not empty." I know what you mean by "and that in itself is not empty" but I think it's worth using more words to say it than to just use "empty". Because as it is, it could be misconstrued because obviously "nothing" is empty as Hegel says. So I think it's worth differentiating empty as the lack of determination and empty as nonsense (which is what I take you to mean).

Reply:
Not sure I follow your issue. The point here is that one could take `nothing` have  `being` _because_ one is thinking and thinking is not empty even if it thinks about nothing (or its thinking is empty). But this approach is wrong because it subjectivizes the logic. Nonsense isn't really part of the discussion here and I don't see how it should be from the text?

Reply:
The distinction of "empty" that I'm referring to comes from the way somebody may misunderstand your uses of it. 

Hegel writes that nothing is complete emptiness - but this logical emptiness is not what you mean when you write that nothing is "in itself not empty" (line 50). But you have used the adjective "empty" to both express the determination of Nothing and to say what Nothing is not, which may be confusing for somebody new to the text.

Reply:
I still don't think it's an issue since I'm using the terms "empty thinking" to distinguish it from the "emptiness" of nothing. 

I have added a sentence to the previous paragraph to help make it clearer (see last commit).